@xen-orchestra/backups/writers/_DeltaBackupWriter.js

```diff
@@ -224,3 +229,7 @@ exports.DeltaBackupWriter = class DeltaBackupWriter {
     // TODO: run cleanup?
   }
 }
+
+Object.assign(DeltaBackupWriter.prototype, abstractExportWriter)
```

Comment:
Ie, it would be best to use `lodash.defaults` here to allow the child class to override parent's methods.